frontend/src/components/social/common/DateBar.jsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect, useState } from "react"
+
+import styled from "styled-components"
+
+import CommonCalendar from "@components/common/CommonCalendar"
+import MildButton from "@components/common/MildButton"
+
+import { useClientLocale } from "@utils/clientSettings"
+import { ifMobile } from "@utils/useScreenType"
+
+import FeatherIcon from "feather-icons-react"
+import { DateTime } from "luxon"
+import { useTranslation } from "react-i18next"
+
+const DateBar = ({ selectedDate, setSelectedDate }) => {
+    const locale = useClientLocale()
+    const { t } = useTranslation("", { keyPrefix: "social.date_bar" })
+
+    // TODO: selectedDate를 Datetime형으로 바꾼 이후 다시 수정
+    const displayDate = () => {
+        const date = DateTime.fromISO(selectedDate)
+        return date.setLocale(locale).toFormat(t("date_format"))
+    }
```

Comment:
날짜 표기를 현지화하고 싶으면 번역 파일에 형식을 지정하는 것보다 `toLocaleString`을 사용하는 것이 좋습니다. [옵션 보기](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat)
```suggestion
            .toLocaleString({
                weekday: "short",
                month: "short",
                day: "numeric",
            })
    }
```